pkg/session_recording/recordings.jsx

```diff
@@ -529,7 +531,13 @@
         }
 
         componentDidMount() {
-            this.journalctlStart();
+            let proc = cockpit.spawn(["getent", "passwd", "tlog"]);
+
+            proc.stream((data) => {
+                this.uid = data.split(":",3)[2];
+                this.journalctlStart();
+            });
+
```

Comment:
Yes, I have not implemented error handling, because it's related to loading animation which I will add also - I'm working on it, in progress.
Actually, about getent I got response from #cockpit, but I will check this.